components/dialogs/formDialog.html

```diff
@@ -12,7 +12,7 @@ <h4 class="modal-title" data-bind="text: title"></h4>
 						<label data-bind="text: name, attr: { for: name }" class="col-lg-2 control-label"></label>
 						<div class="col-lg-10">
 							<input class="form-control" id="inputName"
-								data-bind="value: value, attr: { id: name, placeholder: placeholder, type: type, autofocus: autofocus }">
+								data-bind="value: value, attr: { id: name, placeholder: placeholder, type: type, autofocus: autofocus }"></input>
```

Comment:
Closing `input` tags is not needed because they are [void elements](http://w3c.github.io/html/syntax.html#writing-html-documents-elements).